bertopic/_utils.py

```diff
@@ -147,3 +147,57 @@ def validate_distance_matrix(X, n_samples):
         raise ValueError("Distance matrix cannot contain negative values.")
 
     return X
+
+
+def select_topic_representation(
+    ctfidf_embeddings: Optional[Union[np.ndarray, csr_matrix]] = None,
+    semantic_embeddings: Optional[np.ndarray] = None,
+    use_ctfidf: bool = True
+) -> tuple[np.ndarray, bool]:
+    """Select the topic representation.
+
+    Arguments:
+        ctfidf_embeddings: The c-TF-IDF embedding matrix
+        semantic_embeddings: The semantic embedding matrix
+        use_ctfidf: Whether to use the c-TF-IDF representation. If False, topics embedding representation is used, if it
+                    exists. Default is True.
+    Raises
+        ValueError:
+            - If no topic representation was found
+            - If c-TF-IDF embeddings are not a numpy array or a scipy.sparse.csr_matrix
+
+    Returns:
+        The selected topic representation and a boolean indicating whether it is c-TF-IDF.
+    """
+
+    def to_ndarray(array: Union[np.ndarray, csr_matrix]) -> np.ndarray:
+        if isinstance(array, csr_matrix):
+            return array.toarray()
+        elif not isinstance(array, np.ndarray):
+            raise ValueError("The embeddings should be either a type of `numpy.ndarray` or `scipy.sparse.csr_matrix`")
+        return array
+
+    logger = MyLogger("WARNING")
+
+    if ctfidf_embeddings is None and semantic_embeddings is None:
+        raise ValueError("No topic representation was found.")
```

Comment:
Does this actually happen? Both representations are always saved, so I'm not sure whether it is necessary to check for this.

Reply:
Due to the code readability. Both embeddings can be `None` (by function types). This is the way how to say that one must not be None. 

Not a big deal, I can delete it if you like.

Reply:
As long as this function is not a public function, I think it is not necessary to implement this here. From a communicative perspective, it suggests that there are use cases where you have to check this. As a result, the error message should not only provide the problem but also the solution (or somewhere else, for instance in the docstrings). 

So I would only keep this if there are practical use cases where both might turn out to be `None`.

Reply:
My view is the same as above, but it is ok, I will remove it. 

The problem for me is that I did not see right away from the code (and perhaps this is my problem) that one representation always exists.